.github/workflows/docs-swift.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       - name: Install gmake
         run: brew install make
 
+      - name: Install gsed
+        run: brew install gnu-sed
+
```

Comment:
I would install this when gsed is used in setup-and-cache-rust action

Reply:
But thinking more about it, this is the only github runner that needs the installation so nevermind